sufia-models/lib/sufia/models/stats/user_stat_importer.rb

```diff
@@ -38,8 +38,12 @@ def date_since_last_cache(user)
       end
     end
 
-    def files_for_user(user)
-      ::GenericFile.where(Solrizer.solr_name('depositor', :symbol) => user.user_key)
+    def file_ids_for_user(user)
+      ids = []
+      ::GenericFile.find_in_batches("#{Solrizer.solr_name('depositor', :symbol)}:\"#{user.user_key}\"",{fq:"#{Solrizer.solr_name("has_model", :symbol)}:\"#{::GenericFile.to_class_uri}\"",fl:["id", "#{Solrizer.solr_name('depositor', :symbol)}"]}) do |group|
+        ids.concat group.map{|doc| doc["id"]}
```

Comment:
@mjgiarlo According to ruby docs that code will create an array of arrays: http://www.ruby-doc.org/core-2.2.0/Array.html#method-i-3C-3C
That is definitely not what I want, unless I am missing something.
I could change it to a +, but that returns a new array, which is also not what I want, and slower.
Would you settle for my code with better spacing, or let me know why you code is better?